README.md

```diff
@@ -280,135 +325,104 @@ module.exports = {
 };
 ```
 
-In the example below we are setting the [`browserslist`](#browserslist) to target all the browsers with greater than `0.25%` market share but not IE 11. This information would then be passed to [Autoprefixer](https://github.com/postcss/autoprefixer) to add vendor prefixes and also to [Babel](https://babeljs.io/docs/en) so that it can transpile your Javascript which can be read by all the browsers you wish to support.
-
-### Polyfills
+In the example above we are setting the [`browserslist`](#browserslist) to target all the browsers with greater than `0.25%` market share but not IE 11. This information would then be passed to [Autoprefixer](https://github.com/postcss/autoprefixer) to add vendor prefixes and also to [Babel](https://babeljs.io/docs/en) so that it can transpile your Javascript which can be read by all the browsers you wish to support.
 
-## Deployment environments
+### Polyfilling of newer Javascript features
 
-### Using environment variables
+Underreact polyfills the following Javascript features
 
-Underreact allows you to inject environment variables **during the build time**. You can set them up by creating a `.env` file at the root of your project:
+- `Array.from`
+- `fetch`
+- `Object.assign`
+- `Promise`
+- `Symbol`
 
-```
-#.env
-SERVER=https://example.com
-```
+The above polyfills should allow you to freely use `for..of`, `async function() {}`, `fetch(..)` or `...{}` in your code.
```

Comment:
What is `...{}`?

Reply:
This is applying a spread operator on an Object. Do you want me to use an array?

Reply:
I just couldn't tell from the annotation. We could just say `spread operator`?